ui-v2/app/components/tag-list.js

```diff
@@ -0,0 +1,6 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  tagName: 'dl',
+  classNames: ['tag-list'],
+});
```

Comment:
Tag listing is now being used in various places, so it made sense to make it into a simple component to centralize the markup.

Reply:
big plus one to this, I didn't do this early enough in vault and we now need to make a pass to contain markup for various things and bundle them into components.

Reply:
Yeah there's more to come here. There's a lot in the feature that requires me to reuse things that originally were only in one place but are now in multiple places. I quibbled a lot about what to change in this PR as I didn't want to move too much around. Following on from this there is likely to be a lot of 'lets make that a component now its used in more than one place'.